src/System.Net.Http.Json/pkg/System.Net.Http.Json.pkgproj

```diff
@@ -0,0 +1,10 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Net.Http.Json.csproj">
```

Comment:
We should omit the ref unless we have a good reason to have it.

Reply:
Our model until release/3.1 has been to always have the ref unless it causes issues (like hiding dependencies to RAR when targetting desktop) which this package isn't doing, so in my point of view it would be better to not special-case it and keep it consistent unless we think it would hurt. That said, I'm totally open for suggestions here so I can remove it too if you think we should.

Reply:
We previously did a pass to try to limit the number of places where we expose refs to desktop.  System.Text.Json, for example, does not expose a reference assembly.  All new packages we try not to include refs unless we must have them for some reason.